volkswagencarnet/vw_vehicle.py

```diff
@@ -732,17 +738,8 @@ def last_connected(self) -> str:
 
     @property
     def is_last_connected_supported(self) -> bool:
-        """Return when vehicle was last connected to connect servers."""
-        if next(
-            iter(
-                next(
-                    iter(self.attrs.get("StoredVehicleDataResponse", {}).get("vehicleData", {}).get("data", {})), None
-                ).get("field", {})
-            ),
-            None,
-        ).get("tsCarSentUtc", []):
-            return True
-        return False
+        """Return if when vehicle was last connected to connect servers is supported."""
+        return is_valid_path(self.attrs, "StoredVehicleDataResponse.vehicleData.data.0.field.0.tsCarSentUtc")
```

Comment:
Not 100% sure about this change, but it now checks the presence if the actual value used in the `last_connected` property, so probably okay-ish